edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/TwinsController.cs

```diff
@@ -16,6 +16,8 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Http.Controllers
 
     public class TwinsController : Controller
     {
+        static readonly string supportedContentType = "application/json; charset=utf-8";
```

Comment:
nit: is the spacing after semi-colon is required?

Reply:
the white space is optional according to RFC7231. Most of the examples contain space thought (I copied this one from a google result)